rebar.config

```diff
@@ -12,7 +12,8 @@
        , {docsh,         "0.7.2"}
        , {elvis_core,    "1.1.1"}
        , {rebar3_format, "0.8.2"}
-       , {erlfmt,        "1.0.0"}
+       %%, {erlfmt,        "1.0.0"}
+       , {erlfmt, {git, "https://github.com/gomoripeti/erlfmt.git", {branch, "parser_error_loc"}}} %% Temp until PR is merged
```

Comment:
agree, this PR should only be merged after the erlfmt PR

Reply:
@gomoripeti If this is the only thing left, should we merge this and bump it as a follow up?

Reply:
I should add a hash or tag instead of a branch.
Michal provided some feedback on the PR so my branch might need modifications (and might move)